timeline/templates/timeline/side_timeline.html

```diff
@@ -55,9 +72,16 @@ <h2><i class="iconify" data-icon="mdi:clock-time-eight"></i> {{ timeline_title }
       {% include 'projectroles/_pagination.html' with pg_small=False %}
     {% endif %}
    {% else %}
-    <div class="alert alert-info" role="alert">
-      No site-wide timeline events found{% if timeline_mode == 'object' %} for this object{% endif %}.
-    </div>
+    {% if timeline_mode == 'site' %}
+      <div class="alert alert-info" role="alert">
+        No site-wide timeline events found{% if timeline_mode == 'object' %} for this object{% endif %}.
```

Comment:
It seems this could be shortened into simply having the condition block around the string "site-wide".